webui/master/src/containers/pages/Browse/Browse.test.tsx

```diff
@@ -53,39 +60,57 @@ describe('Browse', () => {
     let shallowWrapper: ShallowWrapper;
 
     beforeAll(() => {
-      shallowWrapper = shallow(<Browse {...props}/>);
+      shallowWrapper = shallow(<BrowsePresenter {...props} />);
     });
 
     it('Renders without crashing', () => {
       expect(shallowWrapper.length).toEqual(1);
     });
 
-    it('Matches snapshot', () => {
-      expect(shallowWrapper).toMatchSnapshot();
-    });
-  });
-
-  describe('App with connected component', () => {
-    let reactWrapper: ReactWrapper;
-
-    beforeAll(() => {
-      reactWrapper = mount(<Provider store={store}><ConnectedApp history={history}/></Provider>);
+    it('Contains a div with class col-12', () => {
+      expect(shallowWrapper.find('.col-12').length).toEqual(1);
     });
 
-    it('Renders without crashing', () => {
-      expect(reactWrapper.length).toEqual(1);
+    describe('Renders Directory Listing', () => {
+      it('Matches snapshot with Table listing', () => {
+        expect(shallowWrapper).toMatchSnapshot();
+      });
     });
 
-    it('Contains the component', () => {
-      expect(reactWrapper.find('.browse-page').length).toEqual(1);
-    });
+    describe('Renders FileView', () => {
+      beforeAll(() => {
+        const data = {...props.browseData};
+        data.currentDirectory.isDirectory = false;
+        shallowWrapper.setProps({browseData: data});
+      });
 
-    it('Calls fetchRequest', () => {
-      sinon.assert.called(props.fetchRequest as SinonSpy);
-    });
-
-    it('Matches snapshot', () => {
-      expect(reactWrapper).toMatchSnapshot();
+      it('Matches snapshot with File', () => {
+        expect(shallowWrapper).toMatchSnapshot();
+      });
     });
   });
+
+  // describe('App with connected component', () => {
+  //   let reactWrapper: ReactWrapper;
+  //
+  //   beforeAll(() => {
+  //     reactWrapper = mount(<Provider store={store}><ConnectedApp history={history}/></Provider>);
+  //   });
+  //
+  //   it('Renders without crashing', () => {
+  //     expect(reactWrapper.length).toEqual(1);
+  //   });
+  //
+  //   it('Contains the component', () => {
+  //     expect(reactWrapper.find('.browse-page').length).toEqual(1);
+  //   });
+  //
+  //   it('Calls fetchRequest', () => {
+  //     sinon.assert.called(props.fetchRequest as SinonSpy);
+  //   });
+  //
+  //   it('Matches snapshot', () => {
+  //     expect(reactWrapper).toMatchSnapshot();
+  //   });
+  // });
```

Comment:
Should we delete this or is this expected to work later? (applies in general to commented out code)

Reply:
cleaned up these commented blocks